ocaml/xapi/import_xva.ml

```diff
@@ -43,7 +43,7 @@ let make __context rpc session_id srid (vms, vdis) =
           (fun _ rpc session_id -> Client.VDI.destroy rpc session_id vdi) :: !clean_up_stack;
         vdi) vdis in
     debug("Now creating all the VMs");
-    let vm_refs = List.map (fun vm ->
+    let make_vm_ref_from_vm = fun vm ->
```

Comment:
The prefix `make` suggests something imperative/generative rather than functional is happening here. Is this the case? Otherwise `vm_ref_from_vm` would be good enough or even `ref_from_vm`.